schematic/store/synapse.py

```diff
@@ -746,22 +746,26 @@ def updateDatasetManifestFiles(
 
         manifest_filepath = self.syn.get(manifest_id).path
         manifest = load_df(manifest_filepath)
+        manifest_is_file_based = "Filename" in manifest.columns
 
-        # update manifest with additional filenames, if any
-        # note that if there is an existing manifest and there are files in the dataset
-        # the columns Filename and entityId are assumed to be present in manifest schema
-        # TODO: use idiomatic panda syntax
+        if manifest_is_file_based:
```

Comment:
@GiaJordan just to make sure that I understand what the expected behavior is, now we change the behavior to avoid adding "file name" and "entity id" columns to all record-based manifest?

Reply:
So the method `fill_in_entity_id_filename` assumes those columns are already present, its really just for updating the manifest with new files that have been added since the last time the manifest was generated. This just avoids calling that if the manifest isn't file based (based on the presence of the `Filename` column existing or not)